javascript/src/main/components/Footer/AppFooter.js

```diff
@@ -15,7 +17,18 @@ const AppFooter = () => {
       <a href="https://github.com/ucsb-cs156-w21/proj-mapache-search" target="_blank" rel="noreferrer noopener">
```

Comment:
Can you update this url to use `s21` rather than `w21` here, so that it's a valid link? You'll likely have to update the corresponding test as well

Reply:
just changed